rts/lra/pom.xml

```diff
@@ -101,6 +104,12 @@
               <groupId>org.sonatype.plugins</groupId>
               <artifactId>nexus-staging-maven-plugin</artifactId>
             </plugin>
+
+<!--            This plugin is used for file logging, when the logging is enabled. But logging is only enabled when CI job is executed.
```

Comment:
I would vote for removing this comment completely as for anybody who understands the `maven` it's superfluous.
If you want to comment then don't say things like "this plugin is used for file logging" which is just not true.
Maybe you can define some property definitions like `test.logs.to.file` but it's still a bit dangerous (by my perspective) to say about default value etc. as those are things which change.

Reply:
Okay, will remove this comment